modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts

```diff
@@ -0,0 +1,181 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {dataSetManagerApiHelpersTest} from './fixtures/dataSetManagerApiHelpersTest';
+import {visualizationModesPageTest} from './fixtures/visualizationModesPageTest';
+import {DEFAULT_LABEL} from './utils/constants';
+
+export const test = mergeTests(
+	dataSetManagerApiHelpersTest,
+	featureFlagsTest({
+		'LPD-10735': true,
+		'LPS-164563': true,
+	}),
+	visualizationModesPageTest,
+	loginTest()
+);
+
+test.beforeEach(async ({dataSetManagerApiHelpers}) => {
+	await dataSetManagerApiHelpers.createDataSet({});
+	await dataSetManagerApiHelpers.createDataSetView({});
+});
```

Comment:
`beforeAll` unfortunately does not work, as we need `page` context, used in `ApiHelpers`.